payment/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.stripe.android.payment">
```

Comment:
What is the actual significance of this value?

Reply:
It's a unique identifier of the module, as stated in the PR comment, `payment-core` is kept `com.stripe.android` to avoid code changes.

When we start adding code to this module, the default root package would be `com.stripe.android.payment`, however it is still possible to add classes to `com.stripe.android`, this would keep our public API package consistent before we move them all to `com.stripe.android.payment`, please see [this section](https://paper.dropbox.com/doc/diagram-Modularizing-Android-Mobile-SDK--BLGnO8aE42lDPYKRlTWlpOerAg-M9mrrTib7RFBVN7VimeGx#:uid=009215492017169343710934&h2=Package-names) for details